programs/CMakeLists.txt

```diff
@@ -201,3 +201,9 @@ endif ()
 if (TARGET clickhouse-server AND TARGET copy-headers)
     add_dependencies(clickhouse-server copy-headers)
 endif ()
+
+if (ENABLE_TESTS AND USE_GTEST)
+    set (CLICKHOUSE_ALL_TESTS_TARGETS local_date_time_comparison unit_tests_libcommon unit_tests_dbms hashing_write_buffer hashing_read_buffer in_join_subqueries_preprocessor expression_analyzer)
+    add_custom_target (clickhouse-tests ALL DEPENDS ${CLICKHOUSE_ALL_TESTS_TARGETS})
```

Comment:
Why do we need this dependency? `clickhouse-bundle` is a target that allows you to get all parts of `clickhouse` to work with it (client, server, benchmark, etc...).

Reply:
It it the only target built for deb
https://github.com/ClickHouse/ClickHouse/blob/master/debian/rules#L30

Reply:
ok